### PR TITLE
chore(operations): Add sync-install to GA

### PR DESCRIPTION
**Pull request by Hoverbear**
Adds missing sync-install to GA for #3001

### CODE REVIEW

**Comment by binarylogic** [comment]
I'm curious what you think about moving this to its own workflow? Where changes to this file on `master` always sync?

**Comment by Hoverbear** [comment]
I'd be totally fine with that too!

**Comment by binarylogic** [comment]
Cool. I think we split it out into a separate GH workflow that is only triggered by changes to `install.sh`.

**Comment by Hoverbear** [comment]
Ok! Will do that!

**Comment by Hoverbear** [comment]
@binarylogic Done, ready for review.

File: .github/workflows/install-sh.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: Update install.sh Suite
+
+on:
+  push:
+    tags:
+      - v0.*
+      - v1.*
```
Comment (binarylogic) [inline .github/workflows/install-sh.yml:7]: Why these tags? Why not sync changes to on `master`?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -6,9 +6,6 @@ on:
       - v0.*
       - v1.*
 
-env:
-  CHANNEL: nightly
```
Comment (binarylogic) [inline .github/workflows/release.yml:10]: Is this intended?

File: .github/workflows/install-sh.yml (hunk 1 above)
Comment (Hoverbear) [reply .github/workflows/install-sh.yml:7]: Oh, yup, missed this.

File: .github/workflows/release.yml (hunk 2 above)
Comment (Hoverbear) [reply .github/workflows/release.yml:10]: I do not see where it is used in https://github.com/timberio/vector/blob/3e4360a2db28b1837e49933e3a6a23b29f97b136/scripts/sync-install.sh, do you know of a reason to keep this?

Comment (Hoverbear) [reply .github/workflows/release.yml:10]: Yup!

**Pull request merged**